account_bank_statement_import_fr_cfonb/wizard/account_bank_statement_import.py

```diff
@@ -158,5 +158,32 @@ def _parse_file(self, data_file):
             'balance_start': start_balance,
             'balance_end_real': end_balance,
             'transactions': transactions,
-            }
+        }
         return currency_code, account_number, [vals_bank_statement]
+
+    def split_lines(self, data_file):
+        """Split the data file into lines.
+
+        Returns a list of the lines in the file provided.
+        """
+        # According to the standard each line has to be 120 chars long, but
+        # some banks ship the files without line break.
+        # so we want to split the file after 120 chars, no matter if there
+        # is a newline there or not.
+
+        # remove linebreaks
+        data_file_without_linebreaks = data_file.replace(
+            '\n', ''
+        ).replace('\r', '')
+
+        # check length of file
+        max_len = len(data_file_without_linebreaks)
+        lines = []
+
+        # make sure the length is a multiple of 120 otherwise it isn't valid
+        if max_len % 120:
```

Comment:
So we are enforcing the CFOND spec here, with maybe generating regressions in current module installations?
Should we fallback on the previous behavior (so splitting lines whatever the size of the lines are)? It is just a question, not blocking. If every banks respect the rule of 120 characters we are fine.

Reply:
Well that is what i got from @CharlineDumontet, i would imagine the 120 chars are more important if there is banks sending the file around without newline. Sadly i can not read the spec as my french is not good enough to navigate the cfonb site to find it (i suspect there might not be a english version of the spec available as well)